change/@ni-nimble-components-92f16ac4-28b7-4ff8-a111-f9a46fbc86aa.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Hey @aagash-ni, @rajsite and I took a quick look at this draft PR to see the direction you're heading. We realized that this infrastructure will require some design work by the Nimble team which we're not ready to devote bandwidth to just yet.

** Near term recommendation **

1. any base tokens which are exposing new color hex values should be added using our existing base token naming conventions (I didn't check if there are any)
2. the SLE calendar integration can continue using base token infrastructure as it does today
3. I will update #2282 to capture concerns with the other parts of this PR, but for now we suggest not adding new base tokens or theme-aware tokens that are specific to the calendar / mobiscroll

** Longer term recommendation **

1. we'll work with @fredvisser to prioritize our team doing the design work
2. once the design work is done we can revive this concept and add the tokens in this PR to nimble (but following the new design) and then update the SLE integration to use them. We'd figure out ownership of that work when it's unblocked.

Hopefully this doesn't derail you too much, but please let us know if you disagree with this direction. And thanks for the draft PR, it was valuable in helping crystallize this proposal.

Reply:
Sure, Jesse.

To proceed with achieving the mockup:

- We'll add the required base tokens with the appropriate naming conventions today.
- Basically the event cards used in the Mobiscroll calendar are templated Angular components. To style these, we'll need the standard theme-aware tokens we typically use.
- For the event cards, I'll introduce a new set of theme-aware tokens (if they don't already exist) based on our needs.
- For calendar styling alone will continue with non theme aware tokens.

I hope this is fine for now to proceed. I'd like to hear your thoughts.

Reply:
That generally seems ok. Regarding
>For the event cards, I'll introduce a new set of theme-aware tokens (if they don't already exist) based on our needs.

We'll just need to make sure these tokens fit well with our conventions. They might be able to follow similar patterns to [graph tokens](https://nimble.ni.dev/storybook/?path=/docs/components-graph--docs) but I'm just less sure of how we'd name them. I'm worried "event card" might too specific of a concept for Nimble but maybe it's ok or maybe we can find a more generic concept. I'd propose you put up a PR (or update this one) with a proposal that follows the [theme aware token naming guidelines](https://github.com/ni/nimble/blob/b814c437bff07dd565ca25facebc0a30e837b428/packages/nimble-components/CONTRIBUTING.md#naming) and we can discuss it in the PR.

Reply:
I have added some essential base tokens (currently unavailable) that are needed to create theme-aware tokens for the calendar. The theme-aware tokens for the calendar will be included in a follow-up PR.

Kindly have a look and let me share your thoughts @jattasNI @rajsite